### DISCUSSION TITLE
What would be the LITE_RPC_HTTP_ADDR url?

### DISCUSSION THREAD

**Discussion by hsripathi** [blockworks-foundation/lite-rpc#390] [Q&A]
Hi,

I'm trying to deploy the lite rpc in fly cloud. I want to replace the default http://0.0.0.0:8890 url. What would be LITE_RPC_HTTP_ADDR replace with. Is this the address of the deployed server machine IP?

**Comment by stevenkader** [comment]
Hi, did you ever get it working.  i am also stuck with the settings to get it working on fly.io.  Thx.

**Reply by hsripathi** [reply]
NO
